apps/web/playwright/fixtures/users.ts

```diff
@@ -334,6 +336,32 @@ const createUser = (workerInfo: WorkerInfo, opts?: CustomUserOpts | null): Prism
   };
 };
 
+async function confirmPendingPayment(page: Page) {
+  await page.waitForURL("/booking/*");
+
+  const url = page.url();
+
+  const params = new URLSearchParams(url.split("?")[1]);
+
+  const id = params.get("payment_intent");
+
+  if (!id) throw new Error(`Payment intent not found in url ${url}`);
+
+  const payload = JSON.stringify({ type: "payment_intent.succeeded", data: { object: { id } } }, null, 2);
+
+  const signature = stripe.webhooks.generateTestHeaderString({
+    payload,
+    secret: process.env.STRIPE_WEBHOOK_SECRET as string,
+  });
+
+  const response = await page.request.post("/api/integrations/stripepayment/webhook", {
+    data: payload,
+    headers: { "stripe-signature": signature },
+  });
+
+  if (response.status() !== 200) throw new Error(`Failed to confirm payment. Response: ${response.text()}`);
+}
```

Comment:
This method it'll be called on /booking/[uid] route, and it'll hit our webhook handler as Stripe it would do in order to confirm that they payment intent succeeded (the charge was done)